WhateverGreen.xcodeproj/project.pbxproj

```diff
@@ -689,7 +694,10 @@
 					"MODULE_VERSION=$(MODULE_VERSION)",
 					"PRODUCT_NAME=$(PRODUCT_NAME)",
 				);
-				HEADER_SEARCH_PATHS = "${PROJECT_DIR}/Lilu.kext/Contents/Resources";
+				HEADER_SEARCH_PATHS = (
+					"${PROJECT_DIR}/Lilu.kext/Contents/Resources",
+					"${PROJECT_DIR}/MacKernelSDK",
```

Comment:
Should not be needed. I do not think any project changes make sense.

Reply:
This include path is removed.